assignments/objects.js

```diff
@@ -18,21 +18,68 @@ const example = {
 }
 
 // Write your intern objects here:
-
+var interns = {
+   Mitzi: {
+	id: 1,
+	email: "mmelloy0@psu.edu",
+	name: "Mitzi",
+	gender: "F"
+   },
+   Kennan: {
+   	id: 2,
+  	email: "kdiben1@tinypic.com",
+  	name: "Kennan",
+  	gender: "M",
+  	speak: function () {
+  		return "Hello, my name is " + this.name;
+  	}
+   },
+   Keven: {
+   	id: 3,
+   	email: "kmummery2@wikimedia.org",
+   	name: "Keven",
+   	gender: "M"
+   },
+   Gannie: {
+   	id: 4,
+   	email: "gmartinson3@illinois.edu",
+   	name: "Gannie",
+   	gender: "M"
+   },
+    Antonietta: {
+   	id: 5,
+   	email: "adaine5@samsung.com",
+   	name: "Antonietta",
+   	gender: "F",
+   	multiplyNums: function (x,y) {
+   		return x*y;
+   	}
+   }	
+}
```

Comment:
Kudos for having your interns' associated action functions inside of their actual object!